lex.go

```diff
@@ -229,15 +237,16 @@ func lexComment(l *lexer) stateFn {
 
 // lexAction consumes a rule action.
 func lexAction(l *lexer) stateFn {
-	r := l.next()
-	switch {
-	case r == ' ':
-		l.emit(itemAction, true)
-		return lexProtocol
-	case unicode.IsLetter(r):
-		return lexAction
+	for {
```

Comment:
This is unrelated refactoring and should have been a separate commit.

Reply:
Do you want to start a new PR without this?

Reply:
It should be fine to leave it in here unless you request some changes.
It's basically the same change applied to the other functions in an earlier PR. I overlooked this one.